rbac/common/sawtooth/rbac_payload.py

```diff
@@ -41,8 +47,37 @@ def make_payload(message, message_type, inputs, outputs):
     )
 
 
+def get_proto(message_type):
+    """Attempts to get the protobuf associated with a message type
+    given the message type name. Can find most protobufs via
+    this method. Used only for testing purposes by unmake.
+    """
+    name = get_message_type_name(message_type)
+    name_parts = name.split("_")
+    try:
+        if len(name_parts) >= 4:
+            # e.g. PROPOSE_REMOVE_ROLE_ADMIN -> role
+            object_type = name_parts[2].lower()
+        else:
+            # e.g. CREATE_USER -> user
+            object_type = name_parts[1].lower()
+
+        proto = getattr(
+            getattr(protobuf, object_type + "_transaction_pb2"),
+            name.title().replace("_", ""),  # camel case the message type name
+        )
+        return proto()
+    except Exception:
+        raise Exception(
+            "get_proto couldn't discover the protobuf for message type {}".format(name)
```

Comment:
We should probably include the underlying cause of the thrown Exception in the new one.